<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTriangle</class>
 <widget class="QMainWindow" name="MainWindowTriangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расчет треугольника</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>       Сторона А</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sideA_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>       Сторона В</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sideB_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>            Угол</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="angleLineEdit_3"/>
     </item>
     <item>
      <widget class="QRadioButton" name="DegreesRadioButton">
       <property name="text">
        <string>Градусы</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RadiansRadioButton">
       <property name="text">
        <string>Радианы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clcSideButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>     Сторона С</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sideC_lineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>286</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Расчет стороны треугольника</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
